### PR TITLE
Retire obsolete offline platform installer page, move prereqs info to ServiceControl page

### PR DESCRIPTION
**Pull request by helenktsai**

### CODE REVIEW

File: servicecontrol/installation.md [hunk 1]
```diff
@@ -22,7 +20,12 @@ NOTE: A [community managed puppet module](https://forge.puppet.com/tragiccode/ns
 
 The ServiceControl installation has the following prerequisites:
 
-* [Microsoft .NET 4.7.2 Runtime](https://dotnet.microsoft.com/download/dotnet-framework/net472)
+* [.NET Framework Runtime 4.7.2 or later](https://dotnet.microsoft.com/download/dotnet-framework/net472).
```
Comment (helenktsai) [inline servicecontrol/installation.md:23]: Is this really a pre-req?

Comment (bording) [reply servicecontrol/installation.md:23]: Yes!

File: platform/installer/index.md
Comment (bording) [inline platform/installer/index.md:1]: We should delete this file as well. In fact, the entire installer folder can go.

File: servicecontrol/installation.md [hunk 2]
```diff
@@ -22,7 +20,12 @@ NOTE: A [community managed puppet module](https://forge.puppet.com/tragiccode/ns
...
+* [.NET Runtime 7.0.4 or later](https://dotnet.microsoft.com/en-us/download/dotnet/7.0).
+* [ASP.NET Core Runtime 7.0.4, or later](https://dotnet.microsoft.com/en-us/download/dotnet/7.0).
+* [Visual C++ 2015 Redistributable](https://www.microsoft.com/en-us/download/details.aspx?id=53840).
+
+These prerequisites will be installed automatically if internet access is available on when installing ServiceControl. However, if the machine does not have internet access, these must be made available manually.
```
Comment (bording) [inline servicecontrol/installation.md:28]:
```suggestion
These prerequisites will be installed automatically if internet access is available installing ServiceControl. However, if the machine does not have internet access, these must be installed manually.
```

**Review by bording — CHANGES_REQUESTED** [review]

**Review by bording — APPROVED** [review]

**Pull request merged**